dev.yml

```diff
@@ -0,0 +1,58 @@
+# Generated by `dev init ios`.
+# Working on a new project? Prefer running the generator to copying this file.
+name: mobile-buy-sdk-android
+
+up:
+  - custom:
+     name: Install Xcode
+     met?:  test -d /Applications/Xcode.app
+     meet: mas install 497799835
+  - custom:
+     name: Symlink Android SDK to ~/Library
+     met?:  test -d ~/Library/Android/sdk
+     meet: mkdir -p ~/Library/Android && ln -snf /usr/local/opt/android-sdk ~/Library/Android/sdk
+  - ruby:
+     package: shopify/shopify/shopify-ruby
+     version: 2.2.3p172-shopify
+  - bundler
+  - android_package:
+    - tools
+    - platform-tools
+    - build-tools-23.0.3
+    - android-23
+    - doc-23
+    - sys-img-x86_64-google_apis-23
+    - addon-google_apis-google-23
+    - extra-android-support
+    - extra-android-m2repository
+    - extra-google-m2repository
+    - android-21
+    - sys-img-x86-android-21
+  - android_emulator:
+      name:                 screenshot
+      android_version:      android-21
+      abi:                  google_apis/x86_64
+      emulator_port:        5560
+      avd_path:             ~/.android/avd/screenshot.avd
+
+commands:
+  ios_test:
```

Comment:
- yervant and sander updated the gradle build.
- I removed the mobiletest gem in exchange for android_emulator
- I changed the first line of the Fastfile to "import_from_git"
- Yervant seems to have copied this dev.yml from mobile-shopify